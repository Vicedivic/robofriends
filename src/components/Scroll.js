import React from "react";

const Scroll = ({ children }) => {
  return (
    <div style={{ overflowY: "scroll", border: "1px solid black", height: 500 }}>
      {children}
    </div>
  );
};

export default Scroll;
