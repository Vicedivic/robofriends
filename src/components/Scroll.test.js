import React from "react";
import ShallowRenderer from "react-shallow-renderer";
import Scroll from "./Scroll";

describe("Scroll", () => {
  const renderer = new ShallowRenderer();
  const scrollHtml = renderer.render(<Scroll />);

  it("renders properly", () => {
    expect(scrollHtml).toMatchSnapshot();
  });
});