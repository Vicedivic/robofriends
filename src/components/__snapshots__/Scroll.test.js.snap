// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll renders properly 1`] = `
<div
  style={
    Object {
      "border": "1px solid black",
      "height": 500,
      "overflowY": "scroll",
    }
  }
/>
`;
